wave-api/src/main/java/io/seqera/wave/api/FusionVersion.java

```diff
@@ -31,6 +31,7 @@
 public class FusionVersion {
 
     static final private Pattern VERSION_REGEX = Pattern.compile(".*/v(\\d+(?:\\.\\w+)*)-(\\w*)\\.json$");
+    static final private Pattern VERSION_REGEX2 = Pattern.compile(".*/pkg\\/(\\d+(?:\\/\\w+)+)\\/fusion-(\\w+)\\.tar\\.gz$");
```

Comment:
Let's call this VERSION_JSON and VERSION_TARGZ

Reply:
done